### PR TITLE
Add doc for Webservice language parameter

### PR DESCRIPTION
**Pull request by jolelievre**
Add doc for Webservice language parameter

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: src/content/1.7/development/webservice/tutorials/advanced-use/multilang-fields.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: Multilingual Fields
+menuTitle: Multilingual
+weight: 6
+---
+
+# Multilingual / localized Fields
+
+Some entities have multilingual / localized fields, byt default the API calls returns the value for ALL languages installed in the Shop, but you might want to fine tune the expected result using the `language` parameter.
```
Comment (Progi1984) [inline src/content/1.7/development/webservice/tutorials/advanced-use/multilang-fields.md:9]:
```suggestion
Some entities have multilingual / localized fields, by default the API calls returns the value for all languages installed in the Shop, but you might want to fine tune the expected result using the `language` parameter.
```

Comment (jolelievre) [reply src/content/1.7/development/webservice/tutorials/advanced-use/multilang-fields.md]: Done

**Review by PierreRambaud — APPROVED** [review]

File: src/content/1.7/development/webservice/tutorials/advanced-use/multilang-fields.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+title: Multilingual Fields
+menuTitle: Multilingual
+weight: 6
+---
+
+# Multilingual / localized Fields
+
+Some entities have multilingual / localized fields, by default the API calls returns the value for all languages installed in the Shop, but you might want to fine tune the expected result using the `language` parameter.
```
Comment (Progi1984) [inline src/content/1.7/development/webservice/tutorials/advanced-use/multilang-fields.md:9]:
```suggestion
Some entities have multilingual / localized fields, by default the API calls return the value for all languages installed in the Shop, but you might want to fine tune the expected result using the `language` parameter.
```

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @jolelievre

**Pull request merged**